cmd/cli/main.go

```diff
@@ -4,57 +4,55 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/woodpecker-ci/woodpecker/cli/drone/build"
-	"github.com/woodpecker-ci/woodpecker/cli/drone/deploy"
-	"github.com/woodpecker-ci/woodpecker/cli/drone/exec"
-	"github.com/woodpecker-ci/woodpecker/cli/drone/info"
-	"github.com/woodpecker-ci/woodpecker/cli/drone/log"
-	"github.com/woodpecker-ci/woodpecker/cli/drone/registry"
-	"github.com/woodpecker-ci/woodpecker/cli/drone/repo"
-	"github.com/woodpecker-ci/woodpecker/cli/drone/secret"
-	"github.com/woodpecker-ci/woodpecker/cli/drone/user"
+	"github.com/woodpecker-ci/woodpecker/cli/build"
+	"github.com/woodpecker-ci/woodpecker/cli/deploy"
+	"github.com/woodpecker-ci/woodpecker/cli/exec"
+	"github.com/woodpecker-ci/woodpecker/cli/info"
+	"github.com/woodpecker-ci/woodpecker/cli/log"
+	"github.com/woodpecker-ci/woodpecker/cli/registry"
+	"github.com/woodpecker-ci/woodpecker/cli/repo"
+	"github.com/woodpecker-ci/woodpecker/cli/secret"
+	"github.com/woodpecker-ci/woodpecker/cli/user"
+	"github.com/woodpecker-ci/woodpecker/version"
 
 	_ "github.com/joho/godotenv/autoload"
 	"github.com/urfave/cli"
 )
 
-// drone version number
-var version string
-
 func main() {
 	app := cli.NewApp()
-	app.Name = "drone"
-	app.Version = version
+	app.Name = "woodpecker"
```

Comment:
```suggestion
	app.Name = "woodpecker cli"
```

Reply:
or `woodpecker-cli`  since the others are called `drone-agent` ... (we should rename them at some point (out of scope of this pull)

Reply:
okay I can change the name, but then we have to change the binary names from `woodpecker` to `woodpecker-cli` as well